server/src/api/user.rs

```diff
@@ -12,54 +12,25 @@ use crate::{
   blocking,
   captcha_espeak_wav_base64,
   websocket::{
-    server::{CaptchaItem, CheckCaptcha, JoinUserRoom, SendAllMessage, SendUserRoomMessage},
-    UserOperation,
+    server::{JoinUserRoom, SendAllMessage, SendUserRoomMessage},
+    UserOperation, WebsocketInfo,
```

Comment:
What's going on with this file here, seems to be using a weirder old version. Repull from main.